includes/class-freemius.php

```diff
@@ -4335,8 +4390,9 @@ function _uninstall_plugin_event( $check_user = true ) {
 			} else {
 				// Send uninstall event.
 				$this->send_install_update( array_merge( $params, array(
-					'is_active'      => false,
-					'is_uninstalled' => true,
+					'is_active'       => false,
+					'is_disconnected' => true,
```

Comment:
@fajardoleo don't change the `is_disconnected` flag on uninstall, we do want to preserve it.